app/src/main/java/comcmput301f16t01/github/carrier/SetLocationsActivity.java

```diff
@@ -229,7 +258,15 @@ public void confirmLocation(View view) {
                 lastBundle.putString("startLocation", new Gson().toJson(locationPoint));
                 Intent intent = new Intent(activity, SetLocationsActivity.class);
                 intent.putExtras(lastBundle);
-                startActivityForResult(intent, PASS_ACTIVITY_BACK);
+                // TODO I know that there is an error here...
+                // if the user is changing the start or end point, we need to grab the duration
+                // and distance so the fare estimate can be updated.
+                // I tried a quick fix that didn't work...it's going to take a fair bit of work to fix it
```

Comment:
Is it a requirement that they change the start and end location? Or is this an extension?